cage/README.md

```diff
@@ -0,0 +1,13 @@
+# Box for Sensor
+There are 2 files for each sensor, box and cover. In the file name it is indicated for which board version each file is.
+
+![alt text](https://github.com/wesliver/SoilLoraSensor/blob/wesliver-patch-1/cage/Wireless_Soil_Moisture_Sensor_3D_Printed_box.png)
```

Comment:
You should do the link relative. You should also put something meaningful in the alt text for people with vision issues.